yarn.lock

```diff
@@ -4753,6 +4753,14 @@
   dependencies:
     tippy.js "^6.3.1"
 
+"@trezor/rollout@1.1.0":
```

Comment:
Why `1.1.0` is installed? In `packages/rollout/package.json` you changed it to `1.2.0`

Reply:
Because of trezor-connect dependency. With new connect version this should be fixed.